src/styles/components/_SearchBar.scss

```diff
@@ -85,21 +85,22 @@
 	top: 100%;
 	left: 0;
 	right: 0;
-	background: $hm-primary-dark;
+	background: rgba( $hm-primary-dark, 0.95 );
 	padding: 0 #{ $gutter-width / 2 };
 	border-radius: 0 0 2px 2px;
 	color: #FFF;
 	margin: 0;
 	list-style: none;
 	z-index: 50;
+	overflow: auto;
 }
 
 .SearchBar_Result {
 
 	@include text-sm;
-	padding: #{ $base-line-height / 2 } 0;
-	margin: 0;
-	border-top: 1px solid lighten( $hm-primary-dark, 8% );
+
+	padding: 0;
+	margin: 0 #{ $gutter-width / -2 };
```

Comment:
Yeah that was intentional... but since you flagged it I can probably remove both that line, and the padding on the container element! Updated.